config/application.rb

```diff
@@ -41,7 +41,8 @@ class Application < Rails::Application
       "Catalog::ApprovalError"             => :service_unavailable,
       "Catalog::SourcesError"              => :service_unavailable,
       "Catalog::RBACError"                 => :service_unavailable,
-      "Discard::DiscardError"              => :unprocessable_entity
+      "Discard::DiscardError"              => :unprocessable_entity,
```

Comment:
@syncrou This should be set to bad request we are not going to be using 422 anymore all 400's